package.json

```diff
@@ -38,7 +40,9 @@
     "image"
   ],
   "dependencies": {
-    "prop-types": "^15.5.8"
+    "prop-types": "^15.5.8",
+    "react": "^15.5.4",
```

Comment:
Ah - I missed that they were in peerDependencies :).  Happy to remove them.  Should we move prop-types there too?